build.sbt

```diff
@@ -51,11 +54,31 @@ libraryDependencies ++= {
     "commons-io" % "commons-io" % "2.5",
     "org.apache.commons" % "commons-lang3" % "3.4",
     "com.github.pathikrit" %% "better-files" % "2.17.1",
+
+    "io.circe" %% "circe-yaml" % "0.6.1",
+
+    "eu.timepit" %% "refined"            % "0.8.2",
+    "com.github.benhutchison" %% "mouse" % "0.8",
+
+    "org.scalatest" %% "scalatest" % "3.0.1" % "test",
+    "com.lihaoyi" %% "ammonite-ops" % "1.0.0-RC7" % "test",
+    "org.scalatest" %% "scalatest" % "3.0.1" % "test",
+
+    "com.beachape" %% "enumeratum" % enumeratumVersion,
```

Comment:
It turns out the parsing works with plain ol' Scala enumerations, so I plan on removing this